app/screens/settings/about/about.tsx

```diff
@@ -238,6 +258,22 @@ const About = ({componentId, config, license}: AboutProps) => {
                         {serverVersion}
                     </Text>
                 </View>
+                {license.IsLicensed === 'true' && loadMetric !== null && (
```

Comment:
Same here.. just in case the answer is yes we should check anyway, it may be better to define a boolean variable that does the check once and avoid the small computation multiple times as well as keeping the code cleaner